crates/chain/src/sqlx_impl.rs

```diff
@@ -0,0 +1,331 @@
+//! Module for stuff
+
+use crate::*;
+use core::str::FromStr;
+
+use alloc::{string::ToString, sync::Arc, vec::Vec};
+use std::prelude::rust_2021::String;
+use bitcoin::consensus::{Decodable, Encodable};
+// use rusqlite;
+use sqlx;
+use sqlx::{Acquire, migrate, Postgres, Row};
+use sqlx::migrate::MigrateError;
+use sqlx::postgres::PgRow;
+// use rusqlite::named_params;
+// use rusqlite::types::{FromSql, FromSqlError, FromSqlResult, ToSql, ToSqlOutput, ValueRef};
+// use rusqlite::OptionalExtension;
+// use rusqlite::Transaction;
+
+/// Table name for schemas.
+pub const SCHEMAS_TABLE_NAME: &str = "bdk_schemas";
+
+
+impl<A> tx_graph::ChangeSet<A>
+where
+    A: Anchor + Clone + Ord + serde::Serialize + serde::de::DeserializeOwned,
+{
+    /// Schema name for [`tx_graph::ChangeSet`].
+    pub const SCHEMA_NAME: &'static str = "bdk_txgraph";
+    /// Name of table that stores full transactions and `last_seen` timestamps.
+    pub const TXS_TABLE_NAME: &'static str = "bdk_txs";
+    /// Name of table that stores floating txouts.
+    pub const TXOUTS_TABLE_NAME: &'static str = "bdk_txouts";
+    /// Name of table that stores [`Anchor`]s.
+    pub const ANCHORS_TABLE_NAME: &'static str = "bdk_anchors";
+
+    /// Initialize sqlite tables.
+    async fn init_postgres_tables(db_tx: &mut sqlx::Transaction<'_, sqlx::Postgres>) -> Result<(), MigrateError> {
+        // let schema_v0: &[&str] = &[
+        //     // full transactions
+        //     &format!(
+        //         "CREATE TABLE IF NOT EXISTS {} ( \
+        // txid TEXT PRIMARY KEY NOT NULL, \
+        // raw_tx BYTEA, \
+        // last_seen BIGINT \
+        // )",
+        //         Self::TXS_TABLE_NAME,
+        //     ),
+        //     // floating txouts
+        //     &format!(
+        //         "CREATE TABLE IF NOT EXISTS {} ( \
+        // txid TEXT NOT NULL, \
+        // vout INTEGER NOT NULL, \
+        // value BIGINT NOT NULL, \
+        // script BYTEA NOT NULL, \
+        // PRIMARY KEY (txid, vout) \
+        // )",
+        //         Self::TXOUTS_TABLE_NAME,
+        //     ),
+        //     // anchors
+        //     &format!(
+        //         "CREATE TABLE IF NOT EXISTS {} ( \
+        // txid TEXT NOT NULL REFERENCES {} (txid), \
+        // block_height INTEGER NOT NULL, \
+        // block_hash TEXT NOT NULL, \
+        // anchor JSONB NOT NULL, \
+        // PRIMARY KEY (txid, block_height, block_hash) \
+        // )",
+        //         Self::ANCHORS_TABLE_NAME,
+        //         Self::TXS_TABLE_NAME,
+        //     ),
+        // ];
+        //
+        // // migrate_schema(db_tx, Self::SCHEMA_NAME, &[schema_v0])
+        // Ok(())
+        // let db = db_tx.acquire().await.unwrap();
+        migrate!("./tx_graph_migrations").run(db_tx).await
```

Comment:
I think you forgot to include these files in the PR.

Reply:
This should no longer be a problem